python_modules/libraries/dagster-k8s/dagster_k8s/job.py

```diff
@@ -714,7 +714,7 @@ def construct_dagster_k8s_job(
     container_config = copy.deepcopy(user_defined_k8s_config.container_config)
 
     if args is not None:
-        container_config["args"] = args
+        container_config["args"] = args + container_config["args"]
```

Comment:
in this approach if we receive `args` passed to `construct_dagster_k8s_job`  and via `container_context`, we combine them allowing one to set tail commands in this way. 

Reasons we may not take this approach:
* potentially breaking for users unexpected leveraging current full override behavior 
* somewhat confusing - no indication in `args` name that it will merge. Existing ignore behavior is also poor so maybe not the strongest argument. 
* gnarly foot gun - not using something like  `&` / `&&` to combine commands likely results in an invalid invocation of `dagster` CLI commands